text/0000-epoch-mascot.md

```diff
@@ -0,0 +1,56 @@
+- Feature Name: epoch_mascot
+- Start Date: 2018-02-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Have a mascot for each epoch.
+
+# Motivation
+[motivation]: #motivation
+
+Branding is an important thing when selling your technology to others, including the non-technical
+or project managers in charge of what gets used. As things change though we also want to be able to
+convey that to users who might not have picked up or used Rust since the last epoch. In this way we
+can have Ferris wear a new costume every epoch, or have an epoch-specific critter mascot much like
+Ubundu does. It also conveys what we mean by [what an epoch is]: Each epoch plays nicely with the
+previous ones despite having breaking changes. In the same way that Ferris has multiple costumes
+they can all be worn, or mixed and matched, and are able to play with each other nicely. On top of
+this they can act as a marker for embodying the accomplishments of the community after each epoch
+begins.
+
+# Detailed design
+[design]: #detailed-design
+
+Every epoch, the community team will look for mascot designs. These can be obtained
+by paying a designer, or from submissions from the community. An idea thread may be put
+up to get ideas for what the "theme" of the mascot will be.
+
+# How We Teach This
+[how-we-teach-this]: #how-we-teach-this
+
+Unlike the more technical RFCs this is one where there would be tribal knowledge
+on the matter and not one that would be documented in places like `TRPL`, or the
+`Nomicon` for instance. Instead it would be taught through engagement with the
+community, having events with the different mascots, as well as things
+like conference T-Shirts.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* Splits up the branding amongst multiple mascots rather than just one.
+
+# Alternatives
+[alternatives]: #alternatives
+
+ - We do nothing at all and continue the course as is without any mascot at all.
+ - We adopt different critters for each epoch, like Ubuntu
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+Should we also be naming the epochs?
```

Comment:
i believe that @rust-lang/core said they would be named by year (she says realizing she is on core). i'll sort this!

Reply:
Sure but they could also have a nick name...

Reply:
Precendent would be something like Android; version 2.3 is "gingerbread" and version 4.0 is "ice cream sandwich"

officially, we've only decided to name them by year, but we could do "code names". that feels like a different RFC in some sense... though i guess maybe not